script/mongoDb/add-network-rpc.ts

```
@@ -0,0 +1,124 @@
+    }
+
+    const client = new MongoClient(MONGODB_URI)
+    await client.connect()
+    const db = client.db('blockchain-configs')
+    const collection = db.collection('RpcEndpoints')
+
+    // Check if there's an existing document for the given chainName
+    const existingDoc = await collection.findOne({ chainName })
+
+    // Check if the RPC endpoint already exists for the given chain
+    if (existingDoc && existingDoc.rpcs) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)